### DISCUSSION TITLE
Specify used version of serverpod_cli in the servers pubspec.yaml.

### DISCUSSION THREAD

**Discussion by fischerscode** [serverpod/serverpod#688] [Ideas]
Melos 3 will have a feature, where the version of the executable is set in the `pubspec.yaml` file as a `dev_dependency`.

This would be a create addition for Serverpod, too.

It would make sure the used version of the `serverpod_cli` command matches the used Serverpod version.

**Comment by vlidholt** [comment]
Didn't know this was possible to do. Interesting!

**Comment by fischerscode** [comment]
It seems like this can be achieved with the [`package:cli_launcher`](https://pub.dev/packages/cli_launcher).
